packages/yourloops/components/device/parameter-change-value.tsx

```diff
@@ -48,7 +48,7 @@ export const ParameterChangeValue: FC<ParameterChangeValueProps> = ({ parameter
       {parameter.previousValue &&
         <>
           <span>{`${formatParameterValue(parameter.previousValue, parameter.previousUnit)} ${parameter.previousUnit}`}</span>
-          <East sx={{ marginInline: theme.spacing(1) }} />
+          <TrendingFlatIcon sx={{ marginInline: theme.spacing(1) }} />
```

Comment:
Was the `East` icon not pretty enough? 😄

Reply:
I found it too big ^^ especially for the dashboard view